draft-ietf-httpbis-http2bis.xml

```diff
@@ -2710,23 +2702,32 @@
         </t>
         <ol spacing="normal" type="1">
           <li>
-              for a response only, zero or more <xref target="HEADERS" format="none">HEADERS</xref> frames (each followed by zero
-              or more <xref target="CONTINUATION" format="none">CONTINUATION</xref> frames) containing the message headers of
-              informational (1xx) HTTP responses (see <xref target="RFC7230" section="3.2"/> and
-              <xref target="RFC7231" section="6.2"/>),
-            </li>
-          <li>
-              one <xref target="HEADERS" format="none">HEADERS</xref> frame (followed by zero or more <xref target="CONTINUATION" format="none">CONTINUATION</xref>
-              frames) containing the message headers (see <xref target="RFC7230" section="3.2"/>),
+              one <xref target="HEADERS" format="none">HEADERS</xref> frame (followed by zero or
+              more <xref target="CONTINUATION" format="none">CONTINUATION</xref> frames) containing
+              the header section (see <xref target="HTTP" section="5"/>),
             </li>
           <li>
-              zero or more <xref target="DATA" format="none">DATA</xref> frames containing the message content (see <xref target="draft-ietf-httpbis-semantics-14" section="6.4"/>), and
+              zero or more <xref target="DATA" format="none">DATA</xref> frames containing the
+              message content (see <xref target="HTTP" section="6.4"/>), and
             </li>
           <li>
-              optionally, one <xref target="HEADERS" format="none">HEADERS</xref> frame, followed by zero or more
-              <xref target="CONTINUATION" format="none">CONTINUATION</xref> frames containing the trailer-part, if present (see <xref target="RFC7230" section="4.1.2"/>).
+              optionally, one <xref target="HEADERS" format="none">HEADERS</xref> frame, followed by
+              zero or more <xref target="CONTINUATION" format="none">CONTINUATION</xref> frames
+              containing the trailer-part, if present (see <xref target="HTTP"
+              section="6.5"/>).
             </li>
         </ol>
+        <t>
+          For a response only, a server MAY send any number of interim responses before the <xref
+          target="HEADERS" format="none">HEADERS</xref> frame containing a final response.  An
+          interim response consists of a <xref target="HEADERS" format="none">HEADERS</xref> frames
+          (which might be followed by zero or more <xref target="CONTINUATION"
+          format="none">CONTINUATION</xref> frames) containing the control data and header section
+          of an interim (1xx) HTTP responses (see <xref target="HTTP" section="15"/>).  A <xref
```

Comment:
Should this reference be more specific, to § 15.2? § 15 is "Status codes", § 15.2 is "Informational 1xx".